src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,362 @@
+			return false
+		},
+		dragOver(e) {
+			if (e.preventDefault) {
+				e.preventDefault()
+			}
+			return false
+		},
+		dragEnter(index, e) {
+			// If it is the last element in the path,
+			// don't do anything
+			if (index === (this.breadcrumbs.length - 1)) {
+				return
+			}
+			// Get the correct element, in case we hover a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.classList && target.classList.contains('crumb')) {
+					const crumbs = document.querySelectorAll('.crumb')
+					crumbs.forEach((f) => { f.classList.remove('over') })
+					target.classList.add('over')
+				}
+			}
+		},
+		dragLeave(e) {
+			// Don't do anything if we leave towards a child element.
+			if (e.target.contains(e.relatedTarget)) {
+				return
+			}
+			// Get the correct element, in case we leave directly from a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.contains(e.relatedTarget)) {
+					return
+				}
+				if (target.classList && target.classList.contains('crumb')) {
+					target.classList.remove('over')
+				}
+			}
+		},
+	},
+	render: function(h) {
+		const breadcrumbs = (this.$slots.default || []).filter(node => !!node && !!node.componentOptions)
+		const crumbs = []
+		// The first half of the breadcrumbs
+		const crumbs1 = this.hiddenIndices.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		crumbs1.forEach((crumb, i) => {
+			if (crumb.elm && crumb.elm.classList) {
+				if (this.isHidden(i)) {
+					crumb.elm.classList.add('hidden')
+				} else {
+					crumb.elm.classList.remove('hidden')
+				}
+			}
+			crumbs.push(crumb)
+		})
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			crumbs.push(h('div', { class: 'crumb svg' },
+				[h(Actions, {
+					class: 'dropdown',
+					props: {
+						forceMenu: true,
+					},
+				}, this.hiddenCrumbs.map(crumb => {
+					return h(ActionRouter, {
+						class: 'crumb',
+						props: {
+							to: crumb.componentOptions.propsData.path,
+							icon: 'icon-folder',
+						},
+					},
+					crumb.componentOptions.propsData.title,
+					)
+				}))]
+			))
+		}
+		// The second half of the breadcrumbs
+		const crumbs2 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(Math.round(breadcrumbs.length / 2))
+			: []
+		crumbs2.forEach((crumb, i) => {
+			if (crumb.elm && crumb.elm.classList) {
+				if (this.isHidden(i + crumbs1.length)) {
+					crumb.elm.classList.add('hidden')
+				} else {
+					crumb.elm.classList.remove('hidden')
+				}
+			}
+			crumbs.push(crumb)
+		})
+
+		return h('div', { class: 'breadcrumb', ref: 'container' }, crumbs)
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+
+.breadcrumb {
+	width: calc(100% - 88px);
+	flex-grow: 1;
+
+	.crumb {
+		&.over {
+			background-color: var(--color-primary-light);
+		}
+
+		&.action-item {
+			ul {
+				overflow-y: auto;
+				-webkit-overflow-scrolling: touch;
+				min-height: calc(44px * 1.5);
```

Comment:
```suggestion
				min-height: calc($clickable-area * 1.5);
```

https://github.com/nextcloud/nextcloud-vue/blob/3a17faff30f290628192f731809f18faefd6d177/src/assets/variables.scss#L26